src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/Application.java

```diff
@@ -2,10 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
+import org.springframework.cloud.openfeign.EnableFeignClients;
 
 @SpringBootApplication
-@EnableCircuitBreaker
+@EnableFeignClients
```

Comment:
just an fyi:

When we come to generate s2s tokens to contact other services this will probably fail as we need to define certain packages e.g:

@EnableFeignClients(basePackages =
    {
        "uk.gov.hmcts.reform.auth",
        "uk.gov.hmcts.reform.authorisation",
        "uk.gov.hmcts.reform.wacaseeventhandler",
    })